src/main/riddl/Venues/Locations/locationTypeDefinitions.riddl

```diff
@@ -0,0 +1,24 @@
+type Status is any of {Draft, Open, Closed} briefly "Internal status of a Location."
+
+type Info is {
+	id: LocationId,
+	name: String,
+  description: String,
+  directions: String briefly "How do you find this location?",
+  latLong: LatLong,
+  url: URL?,
+  capacity: Integer?,
+  accessible: Boolean briefly "Is this location handicapped accessible?",
+  amenities: String*, //TODO: We may need amenities to be searchable - i.e. I need to find a Location that is handicaped accessible and has a kitchen
```

Comment:
Should amenities be an enumerated list of strings? Searching becomes harder if somebody misspells stuff (accessalbe, Kithcen, etc.)

Reply:
Amenities will probably need to be their own entities. This is left over from the original VenueContext that had Location and Reservation in the same file as Venue. I can get rid of it until we see Amenities being worth building out